src/components/views/rooms/RoomBreadcrumbs.js

```diff
@@ -81,6 +86,13 @@ export default class RoomBreadcrumbs extends React.Component {
         }
     }
 
+    onMyMembership = (room, membership) => {
+        if (membership === "leave" || membership === "ban") {
+            // Force left rooms to render appropriately
+            this.forceUpdate();
```

Comment:
Can we avoid `forceUpdate()` here please? E.g. have a `hasLeft: true` prop or something in `this.state.rooms` and render based on that...